content/en-us/tutorials/core/building/create-basic-visual-effects.md

```diff
@@ -156,7 +156,7 @@ To create dust particles:
    </thead>
    <tbody>
    <tr>
-   <td>VFX_DustMotes</td>
+   <td>Emitter_DustMotes</td>
```

Comment:
This naming convention is similarly valid, but a different route from the sample place file.